plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/FBTImporter.java

```diff
@@ -962,8 +962,8 @@ private void parseInternalFB(final BaseFBType type) throws TypeImportException,
 		final String typeFbElement = getAttributeValue(LibraryElementTags.TYPE_ATTRIBUTE);
 		TypeEntry entry = getTypeEntry(typeFbElement);
 		if (entry == null) {
-			entry = addDependency(
-					getTypeLibrary().createErrorTypeEntry(typeFbElement, LibraryElementPackage.eINSTANCE.getFBType()));
+			entry = addDependency(getTypeLibrary().createErrorTypeEntry(typeFbElement == null ? "" : typeFbElement, //$NON-NLS-1$
```

Comment:
I suggest moving the null check to the beggining of https://github.com/eclipse-4diac/4diac-ide/blob/819cb6252b9688ab77d2530f7aa3333b54d770a7/plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/typelibrary/TypeLibrary.java#L250 in order to catch all potential cases, present or future.